### PR TITLE
fix : #1542 formatting whoami to who-am-i

### LINKED ISSUE #1542: [Bug]: formatting "whoami" to "who-am-i"
### Describe the bug

In the terminal commands are "sign-in", "sign-out", "get-location" in kebab-case expect "whoami" it's flat case, "whoami" is converted into flat to kebab-case.

### To Reproduce

1. Go to terminal component
2. Click on "whoami"
3. See error
4. change "whoami" to "who-am-i"

### Expected behavior

image

"whoami" to "who-am-i"

### Screenshots/Videos

Screenshot 2024-07-03 204740

### Additional context

_No response_

### Please checkmark the following checklist

- [x] I make sure that similar issue is not opened

### PR DESCRIPTION
**Pull request by ADITYAMAURYA1**
## formatting whoami to who-am-i

Fixes #1542 

image

the pr acomplish this issue #1542

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.multiRootWorkspaceName": "piyushgargdev-nextjs"
```
Comment (sujal12344) [inline .vscode/settings.json:2]: why this?

File: yarn.lock [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "@aws-sdk/util-utf8-browser" "^3.0.0"
     tslib "^1.11.1"
 
-"@aws-crypto/sha256-js@^3.0.0", "@aws-crypto/sha256-js@3.0.0":
+"@aws-crypto/sha256-js@3.0.0", "@aws-crypto/sha256-js@^3.0.0":
```
Comment (sujal12344) [inline yarn.lock:40]: what are you do in .lock file?

File: yarn.lock [hunk 3]
```diff
@@ -607,7 +607,7 @@
     "@aws-sdk/types" "3.347.0"
     tslib "^2.5.0"
 
-"@aws-sdk/types@^3.222.0", "@aws-sdk/types@3.347.0":
+"@aws-sdk/types@3.347.0", "@aws-sdk/types@^3.222.0":
```
Comment (sujal12344) [inline yarn.lock:610]: here also

**Comment by sujal12344** [comment]
@ADITYAMAURYA1 please provide video.

**Comment by ADITYAMAURYA1** [comment]
It is a mistake I added all the local evvironment setup project . and created the pr

let me correct this

**Pull request closed without merging**